packages/x-codemod/src/v6.0.0/pickers/replace-toolbar-props-by-slot/actual.spec.js

```diff
@@ -0,0 +1,7 @@
+<DatePicker ToolbarComponent={MyToolbar} />;
```

Comment:
We could also consider adding even more props to make sure the previous ones are left intact. 🤔 
```suggestion
<DatePicker
  ToolbarComponent={MyToolbar}
  showToolbar={false}
  components={{
    ActionBar: CustomActionBar,
  }}
  componentsProps={{
    actionBar: {
      actions: [],
    },
  }} />;
```